december-2017/forum-structure-changes.md

```diff
@@ -0,0 +1,12 @@
+As part of [our ongoing efforts to streamline and concentrate development](https://blog.mybb.com/2017/10/21/a-fresh-perspective/), we have recently implemented some changes to the structure of the forums used to gather suggestions and feedback on the current and future direction of the MyBB platform.
+
+Previously we had separate suggestion and feedback forums for each different development branch. This led to there being quite a lot of duplicates between suggestions for 1.8, 1.10 and 2.0. In order to try and clarify the future direction of development and make it easier for everybody to follow, we have decided to consolidate all suggestions and feedback into a single forum: [Suggestions and Feedback](https://blog.mybb.com/2017/10/21/a-fresh-perspective/).
+
+This new forum has a set of thread prefixes to specify versions that a suggesiton or piece of feedback is targetted at to try and make it easier to read through. It also contains two subforums:
+
+- [Planned Features](https://community.mybb.com/forum-194.html) - this forum is used for any features that have been confirmed to be part of a future release. Some of the threads here may be current works in progress, or may even be fully implemented already.
+- [Rejected Features](https://community.mybb.com/forum-195.html) - this forum is used for any features that have been rejected. This could be due to the request being a duplicate of another planned feature, or due to it being outside of the scope of MyBB's current direction.
+
+This restructuring is still currently a work in progress, and as such the old [1.8 Suggestions and Feedback](https://community.mybb.com/forum-158.html) forum still exists, though new threads cannot be created there. The same goes for the [2.0 Suggestions and Feedback forum](https://community.mybb.com/forum-152.html).
+
+As ever, we welcome any feedback on this new structure and are always lookin for ways to improve visibility into the direction that we are heading. We hope that by restructuring in these kinds of way the future of development should become more clear.
```

Comment:
As ever, we welcome any feedback on this new structure and are always looking for ways to improve visibility into the direction that MyBB is heading. 

Slight spelling mistake.